libbpf-tools/tcpretrans.bpf.c

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Anton Protopopov
+// Copyright (c) 2021 Red Hat
+//
+// Based on tcpconnect.c by Anton Protopopov and
+// tcpretrans(8) from BCC by Brendan Gregg
```

Comment:
C style comment.

Reply:
Many files in this directory follow this pattern.

Reply:
Many review comments about code style I posted here was proposed before by maintainers/reviewers of this repo. I think is good to follow those rules which mainly comes from kernel code style guide.

Recent commits added by me all use C style comments which is suggeted by Andrii.